python/activator/middleware_interface.py

```diff
@@ -36,7 +36,7 @@
     FLAT="flat.yaml",
 )
 
-_log = logging.getLogger(__name__)
+_log = logging.getLogger("lsst." + __name__)
```

Comment:
This uses `_log` here, but `logger` in `activator.py`; we should probably make those consistent, but I don't have a strong preference for one or the other (maybe `logger` because it doesn't require typing a `_`?).

Reply:
The underscore at least makes it clear that it's a private package variable.

Reply:
I'll change it to `_log`.